components/notebook-controller/controllers/notebook_controller.go

```diff
@@ -241,19 +241,15 @@ func (r *NotebookReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 				if pod.Status.ContainerStatuses[i].State == instance.Status.ContainerState {
 					continue
 				}
-
-				log.Info("Updating Notebook CR state: ", "namespace", instance.Namespace, "name", instance.Name)
-				cs := pod.Status.ContainerStatuses[i].State
-				instance.Status.ContainerState = cs
```

Comment:
Lastly, let's keep this logic that updates the `instance.Status.ContainerState` of the CR. The web app is using this information.

Reply:
Agree